oracledb-native/src/main/java/org/ballerinalang/oracledb/nativeimpl/ClientProcessor.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+ package org.ballerinalang.oracledb.nativeimpl;
+
+import io.ballerina.runtime.api.values.BMap;
+import io.ballerina.runtime.api.values.BObject;
+import io.ballerina.runtime.api.values.BString;
+import org.ballerinalang.oracledb.Constants;
+import org.ballerinalang.oracledb.utils.Utils;
+import org.ballerinalang.sql.datasource.SQLDatasource;
+
+import java.util.Properties;
+
+/**
+ * This class contains the utility methods for the oracledb clients.
+ *
+ * @since 0.1.0
+ */
+public class ClientProcessor {
```

Comment:
If this class contains utility functions, shall we rename the class as `ClientProcessorUtils`

Reply:
ClentProcessor has the logic to process the connection parameters itself. It does not contain any methods that are used by other methods as utilities. Shall I change the class comment instead?

Reply:
Changed the documentation to not have the word utility with f203f1b. Will change the name of the class if you think `ClientProcessorUtils` is more appropriate

Reply:
@anupama-pathirage WDYT?